lib/screens/event/screen.dart

```diff
@@ -20,14 +21,14 @@ class _EventsScreenState extends State<EventsScreen> with Screen {
       GlobalKey<RefreshIndicatorState>();
 
   StreamController dropdownMenuStreamController = StreamController.broadcast();
-  Stream<List<EventCard>> stream;
+  Stream<List<Event>> stream;
   int userUid;
 
   Future<Null> load() async {
     final prefs = await SharedPreferences.getInstance();
     final eventsSortOption = prefs.getString('eventsSortOption');
```

Comment:
change connectSharedPrefs to take in the key; ex: connectSharedPrefs('eventsSortOption'), so that some code like "final prefs = await SharedPreferences.getInstance();" isnt typed twice